Updated Readme (#1)
WayScience/CFReT-screen-mAP-analysis: README.md

This pathological process is characterized by the excessive deposition of extracellular matrix (ECM) components, especially collagen, resulting in the formation of scar tissue.
While ECM production is a vital part of the body’s response to injury, its overproduction disrupts tissue homeostasis and leads to structural changes that impair organ function.
In the heart, for example, excessive ECM accumulation causes stiffening of the myocardial tissue, compromising its ability to contract and relax efficiently.
This can lead to reduced cardiac output, impaired electrical conductivity, and an increased risk of heart failure or fatal arrhythmias.
Given its global impact and contribution to high mortality rates, there is an urgent need to develop advanced diagnostic tools and targeted therapeutic strategies to better understand, manage, and treat cardiac fibrosis.

Recently, we applied image-based profiling with the [CellPainting](https://www.nature.com/articles/nprot.2016.105)(CP) assay to investigate whether single-cell image-based profiles could be leveraged to identify failing heart cells (cardiac fibrosis), healthy heart cells, and heart cells treated with `drug_x`.

> Our study demonstrated that by using image-based profiles into machine learning models, we could accurately distinguish between failing and healthy cells.

Reviewer: ```suggestion
Our study demonstrated that by using image-based profiles with machine learning models, we could accurately distinguish between single cells derived from either heart failure or non-heart failure patients.
```

> Additionally, we identified morphological features that were critical for the model’s ability to differentiate between cell states.

Reviewer: ```suggestion
Additionally, we identified morphological features that were critical for the model’s ability to differentiate between heart states.
```

Reviewer: Recommend not stating that we are looking at cell state since there is a lot of heterogeneity within these cell populations per heart. We are predicting if these cells from heart failure or "healthy" hearts.

These findings suggest that specific cellular structures are strong indicators of heart cell health or failure.
The processing methods and results of this analysis can be accessed [here](https://github.com/WayScience/cellpainting_predicts_cardiac_fibrosis).

> Building upon these  results, we advanced our study to the next level by leveraging high-content screening (HCS) with CP assay. Specifically, we applied a library of 550 small-molecule compounds to identify those with potential to reverse the effects of cardiac fibrosis.

Reviewer: ```suggestion
Building upon these results, we advanced our study to the next level by leveraging high-content screening (HCS) with CP assay. 
Specifically, we applied a library of 550 small-molecule compounds to identify those with the potential to reverse the effects of cardiac fibrosis.
```
